.github/workflows/shippable_builds.yml

```diff
@@ -744,6 +778,8 @@ jobs:
           fail_on_unmatched_files: true
           files: |
             uploads/${{ steps.rename.outputs.PKG_FILE }}
+            uploads/${{ steps.pkginfo.outputs.TAG_NAME }}.tar.gz
+            uploads/${{ steps.pkginfo.outputs.TAG_NAME }}.zip
```

Comment:
From what I understand, you can upload your own zip and tar files. I've not tested this, so I don't know what the behavior will be. The reason for generating our own is because we have to now upload them to FTP, so there are 2 options that I can think of: 1) download what github has generated, or 2) generate them on our own and upload to github/ftp.

Reply:
Ok I'll plan to generate them for FTP upload only